### PR TITLE
workflow: remove GerritDSL

### PR DESCRIPTION
**Pull request by alonbl**
GerritDSL not documented and it seems redundant to current steps. Other
trigger plugins does not seem to provide this functionality.

Duplicate logic should be avoided if possible.

Signed-off-by: Alon Bar-Lev <alon.barlev@gmail.com>

### CODE REVIEW

**Comment by alonbl** [comment]
> Should we have a note in the README.md for that?

Sure if it is usable, but you gave an example of DSL that reuse the existing steps, why is this required?

**Comment by lucamilanesio** [comment]
> > Should we have a note in the README.md for that?
> 
> Sure if it is usable, but you gave an example of DSL that reuse the existing steps, why is this required?

Just in case someone is upgrading from v0.2.x: just a small note would suffice and how to migrate to the new steps.

**Comment by alonbl** [comment]
On Sun, Oct 28, 2018 at 11:44 AM Luca Milanesio <notifications@github.com>
wrote:

> Should we have a note in the README.md for that?
>
> Sure if it is usable, but you gave an example of DSL that reuse the
> existing steps, why is this required?
>
> Just in case someone is upgrading from v0.2.x: just a small note would
> suffice and how to migrate to the new steps.
>
>
OK, although I have no idea how it was used if not documented :) :)

File: README.md [hunk 1]
```diff
@@ -102,6 +102,8 @@ Parameters:
 
 ### Declarative pipeline example
 
+> Note: the gerrit DSL helper was removed in 0.3, please use the following.
+
```
Comment (alonbl) [inline README.md]: done

**Pull request merged**